examples/otel-local-config.yaml

```diff
@@ -0,0 +1,59 @@
+   extensions:
+     health_check:
+       endpoint: 0.0.0.0:13133
+     pprof:
+       endpoint: 0.0.0.0:1777
+     zpages:
+       endpoint: 0.0.0.0:55679
+
+   receivers:
+
+     opencensus:
+       endpoint: 0.0.0.0:55678
+
+     # Collect own metrics
+     prometheus:
+       config:
+         scrape_configs:
+         - job_name: 'otel-collector'
+           scrape_interval: 10s
+           static_configs:
+           - targets: ['0.0.0.0:8888']
+
+     jaeger:
+       protocols:
+         grpc:
+           endpoint: 0.0.0.0:14250
+         thrift_binary:
+           endpoint: 0.0.0.0:6832
+         thrift_compact:
+           endpoint: 0.0.0.0:6831
+         thrift_http:
+           endpoint: 0.0.0.0:14268
+         thrift_tchannel:
```

Comment:
side note: soon to be moved to contrib, just to keep an eye to coordinate and do not let it go out of sync